charts/shoot-core/components/charts/monitoring/charts/node-exporter/templates/node-exporter.yaml

```diff
@@ -83,7 +83,7 @@ spec:
         - --collector.diskstats
         - --collector.filefd
         - --collector.filesystem
-        - --collector.filesystem.mount-points-exclude=^/.+$
+        - --collector.filesystem.mount-points-exclude=^/(run|var)/.+$|^/(boot|dev|sys|usr)($|/.+$)
```

Comment:
If I understand this regex correctly, we want to include `/run` and exclude everything i.e. `/run/*` ?

Reply:
Exactly.
There are many subpaths (for each container) that we'd like to ignore: `/run/containerd/io.containerd.runtime.v2.task/k8s.io/6335e3a8d7cb5542205cac258f3098fd7cebc454f765a6db9120670e2adc0e66/rootf`.

Motivation: the `/run` mount can fill up with empty directories. See https://github.tools.sap/kubernetes/landscape-issues/issues/62